packages/nimble-components/src/combobox/template.ts

```diff
@@ -76,7 +76,7 @@ ComboboxOptions
                 ?disabled="${x => x.disabled}"
                 ${ref('listbox')}
             >
-                <slot
+                <slot name="options"
```

Comment:
Can you update the PR description to explain this change (and the corresponding one in mention-listbox)? And just to double check, this isn't client-facing, right? Clients can still slot combobox options in the default slot?

Reply:
Updated the description. And for posterity, yes, this isn't client-facing. A client can still slot options into the default slot.